### ISSUE TITLE
Reification popups

### ISSUE THREAD

**Issue by jamesamcl** [EBISPOT/ols4#225]
While it's great that we have access to the reification data, the UI isn't great:

* The font size is very small
* Multiple values are munged together
* Links aren't clickable
* IRIs aren't mapped to to identifiers using `linkedEntities` as they would be in the main page

<img width="613" alt="Screenshot 2023-02-24 at 02 20 05" src="https://user-images.githubusercontent.com/454895/221076118-9c0e338a-2a63-47a7-896c-0bc0fce23800.png">

<img width="365" alt="Screenshot 2023-02-24 at 02 21 16" src="https://user-images.githubusercontent.com/454895/221076286-adf4e929-42ef-4fb5-8a23-321f3ff67a34.png">

I think we need a much bigger and grander popup using the same elements we use on the page.

**Comment by matentzn** [comment]
Bigger is good but why not curiefy as well?

**Comment by jamesamcl** [comment]
I agree - lots of places we can do that so I've opened it as another issue #227

**Comment by haideriqbal** [comment]
Closing this due to inactivity. Feel free to reopen
